### DISCUSSION TITLE
DISTINCTCOUNT Function help!

### DISCUSSION THREAD

**Discussion by eb1064** [DaxStudio/DaxStudio#689] [Q&A]
I am relatively new to DAX Studio. I have a .csv with over 2 million rows. I opened the data in Excel and loaded it to a Power Query model. I have a column (gr) that has numbers 1-67, with many duplicates. I need to calculate the number of occurrences for each number.

I would like the output to look like this:

1    5000
2    4561
3    8764
4    4781
...

Where it shows the number (1-67) in one column and the number of occurrences in the other. I think I can use the DISTINCTCOUNT function, but I don't know the proper syntax. Please help!

Attached for reference is a sample of roughly 4,000 lines of my data.
[Sample.xlsx](https://github.com/DaxStudio/DaxStudio/files/6903895/Sample.xlsx)

**Comment by dgosbell** [comment]
This does not sound like a distinctcount problem, It sounds more like you just want to group by the gr column and count the rows.

You could build this in the Query Builder by dragging the (gr) column into the Columns/Measures area. Then click the **new** button at the top to create a new measure within the builder, then enter the following expression `COUNTROWS( <your table name> )`

This should produce similar output to writing a query like the following:
```
EVALUATE
SUMMARIZECOLUMNS(
<your table name>[gr],
"Count",
CALCULATE( COUNTROWS( <your table name> ) )
)
```
